caddyhttp/httpserver/server.go

```diff
@@ -35,13 +35,11 @@ import (
 	"github.com/caddyserver/caddy/caddyhttp/staticfiles"
 	"github.com/caddyserver/caddy/caddytls"
 	"github.com/caddyserver/caddy/telemetry"
-	"github.com/lucas-clemente/quic-go/http3"
```

Comment:
They still have this dependency in the master branch of the caddy repo. They say it supports go1.18 or higher? It was breaking for me too when I tested it but trying to figure out why now...

Reply:
We are not making use of this functionality in our caddy config so there is no harm in removing it